packages/perseus-editor/src/components/locked-point-settings.tsx

```diff
@@ -18,13 +19,24 @@ import {getValidNumberFromString} from "./util";
 
 import type {LockedPoint} from "@khanacademy/perseus";
 
+const possibleColors = [
+    "purple",
+    "blue",
+    "teal",
+    "green",
+    "gold",
+    "red",
+    "pink",
+];
```

Comment:
These are not the exact colors from the [designs](https://www.figma.com/file/JroPokeCLBv2VRIw3GPvY5/LC---Interactive-graph-widget?type=design&node-id=739-6745&mode=design&t=9dixLbZO1PneX7r0-0), but I'm not sure where those are? I did my best to match them with Wonder Blocks colors.

Reply:
I think there's a similar list defined in the `mathjax-renderer` repo.  You might want to work with Ben and/or Tamara to see if that list should be defined somewhere centrally?